daemon/src/setup_maker.rs

```diff
@@ -101,6 +101,7 @@ impl Actor {
             self.build_party_params.clone_channel(),
             self.sign.clone_channel(),
             Role::Maker,
+            self.order.position_maker,
```

Comment:
To be consistent, you could also get the position from `start_contract_setup` here instead of passing it in through the `order.` Not sure what is cleaner.

Reply:
It's simpler and more consistent that way because we don't have to pass any extra data to the `setup_taker.rs`.